domain/topn_slow_query.go

```diff
@@ -40,11 +43,11 @@ func (h *slowQueryHeap) Pop() interface{} {
 	return x
 }
 
-func (h *slowQueryHeap) Refresh(now time.Time, recent time.Duration) {
+func (h *slowQueryHeap) Refresh(now time.Time, period time.Duration) {
```

Comment:
Why it is exported?

Reply:
This is not exported, `slowQueryHeap` is in lower case